main.js

```diff
@@ -23,8 +65,19 @@
             const records = data.result.records;
             records.forEach((record, i) => {
                 console.log(i);
-                L.marker([record.Y, record.X]).addTo(map)
+
+                //Collect time of incident from the record
+                var incidentYear = parseInt(record.INCIDENTTIME.substring(0,4));
+                var incidentMonth = parseInt(record.INCIDENTTIME.substring(6,8));
+                var incidentDay = parseInt(record.INCIDENTTIME.substring(8,10));
+
+                const marker = L.marker([record.Y, record.X]);
```

Comment:
This would become `record.pin = L.marker([record.Y, record.X]);` in the new system